### PR TITLE
Unit tests elrond hyper block endpoint

### PR DESCRIPTION
**Pull request by mariusmihaic**
- Added tests for `elrondHyperBlockEndPoint`
- Adapted file naming to the project using snake_case

### CODE REVIEW

File: api/elrond_hyper_block_endpoint_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/ElrondNetwork/covalent-indexer-go/hyperBlock"
+	"github.com/ElrondNetwork/covalent-indexer-go/testscommon/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestElrondHyperBlockEndPoint_GetHyperBlock(t *testing.T) {
```
Comment (sstanculeanu) [inline api/elrond_hyper_block_endpoint_test.go:39]: coverage 100% can be easily achieved with 2 more tests, one with nil on constructor and another one with `ReadCloserStub` returning error on `Close`

**Review by ssd04 — COMMENTED** [review]
no other comments from my side

File: api/elrond_hyper_block_endpoint_test.go (hunk 1 above)
Comment (mariusmihaic) [reply api/elrond_hyper_block_endpoint_test.go:39]: Added tests for 100% coverage

**Review by ssd04 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]
